apps/backend/src/main/java/no/nav/data/team/dashboard/DashboardController.java

```diff
@@ -57,19 +63,27 @@
     private final TeamService teamService;
     private final ClusterService clusterService;
     private final NomClient nomClient;
-    private final LoadingCache<String, DashResponse> dashData;
+
+    @Autowired
+    private LoadingCache<String, DashResponse> getDashCache;
 
     private static final List<Team> E = List.of();
     private static final TreeSet<Integer> groups = new TreeSet<>(Set.of(0, 5, 10, 20, Integer.MAX_VALUE));
     private static final TreeSet<Integer> extPercentGroups = new TreeSet<>(Set.of(0, 25, 50, 75, 100));
     private static final BiFunction<Object, Integer, Integer> counter = (k, v) -> v == null ? 1 : v + 1;
 
+    @Autowired
     public DashboardController(ProductAreaService productAreaService, TeamService teamService, ClusterService clusterService, NomClient nomClient) {
         this.productAreaService = productAreaService;
         this.teamService = teamService;
         this.clusterService = clusterService;
         this.nomClient = nomClient;
-        this.dashData = Caffeine.newBuilder()
+
+    }
+
+    @Bean(name="getDashCache")
```

Comment:
Trenger dette være en bean?

Reply:
Dette var workaround for å kunne invalidere cache i "afterTest"

Reply:
Kunne muliggens vært bedre med en cache-service klasse.

Reply:
Jeg ville nok splittet det opp i controller - egen bean for cache og en service klasse for oppbygging av statetstikk. Men det er egen smaksak hvordan man liker å dele opp kode :) Så lenge det er lesbart og ikke bryter funksjonelt er jeg glad